apps/backend/src/api/vms.py

```
@@ -24,8 +27,8 @@ async def get_vm_logs(
     hostname: str = Path(..., description="Device hostname"),
     timeout: int = Query(30, description="SSH timeout in seconds"),
     live: bool = Query(False, description="Force fresh data collection"),
-    current_user=Depends(get_current_user),
-):
+    current_user: Any = Depends(get_current_user),
+) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)